### PR TITLE
feat: Add AdditionalVolumes and AdditionalVolumeMounts to workround statufulset 

### PR DESCRIPTION
**Pull request by jiuker**
fix: https://github.com/minio/operator/issues/1710
Add AdditionalVolumes and AdditionalVolumeMounts to workround statufulset
How to test:

1. deploy minio-operator ready
2. create a secret
3. deploy the tenant with the secret mounted.

### CODE REVIEW

File: docs/tenant_crd.adoc [hunk 1]
```diff
@@ -804,6 +808,18 @@ TenantSpec (`spec`) defines the configuration of a MinIO Tenant object. +
 |*Optional* + 
  Specify a secret that contains additional environment variable configurations to be used for the MinIO pools. The secret is expected to have a key named config.env containing all exported environment variables for MinIO+
 
+|*`initContainers`* __link:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#container-v1-core[$$Container$$] array__ 
+|*Optional* + 
+ Add customs initContainers to StatefulSet
```
Comment (shtripat) [inline docs/tenant_crd.adoc:813]: Should it be `custom` and not `customs`?

File: docs/tenant_crd.adoc [hunk 2]
```diff
@@ -804,6 +808,18 @@ TenantSpec (`spec`) defines the configuration of a MinIO Tenant object. +
...
+
+|*`additionalVolumes`* __link:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#volume-v1-core[$$Volume$$] array__ 
+|*Optional* + 
+ If provided, work statefulset will add this volumes. You should set the rules for the corresponding volume and VolumeMount. We will not test this rule, K8S will show the result.
```
Comment (shtripat) [inline docs/tenant_crd.adoc:817]: Spell and grammar check
```
If provided, statefulset will add these volumes. You should set the rules for the corresponding volumes and volume mounts. We will not test this rule, k8s will show the result.
```

File: docs/tenant_crd.adoc [hunk 3]
```diff
@@ -804,6 +808,18 @@ TenantSpec (`spec`) defines the configuration of a MinIO Tenant object. +
...
+
+|*`additionalVolumeMounts`* __link:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#volumemount-v1-core[$$VolumeMount$$] array__ 
+|*Optional* + 
+ If provided, work statefulset will add this volumemunts. You should set the rules for the corresponding volume and VolumeMount. We will not test this rule, K8S will show the result.
```
Comment (shtripat) [inline docs/tenant_crd.adoc:821]:
```
If provided, statefulset will add these volume mounts. You should set the rules for the corresponding volumes and volume mounts. We will not test this rule,  k8s will show the result.
```

File: docs/tenant_crd.adoc (hunk 2 above)
Comment (jiuker) [reply docs/tenant_crd.adoc]: applied the suggestion. Thanks @shtripat

File: docs/tenant_crd.adoc (hunk 3 above)
Comment (jiuker) [reply docs/tenant_crd.adoc]: applied the suggestion. Thanks @shtripat

**Review by harshavardhana — APPROVED** [review]

**Review by shtripat — APPROVED** [review]
lgtm

**Pull request merged**